LICENSE.md

```diff
@@ -27,6 +27,10 @@ for exceptions.
 > OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 > WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+Julia includes code from the following projects, which have their own licenses:
+- [LDC](https://github.com/ldc-developers/ldc/blob/master/LICENSE) (for ccall/cfunction ABI definitions)
+- [MUSL](http://git.musl-libc.org/cgit/musl/tree/COPYRIGHT) (for getopt implementations on Windows)
+- [NetBSD](http://www.netbsd.org/about/redistribution.html) (for setjmp/longjmp implementations on Windows)
```

Comment:
If the MUSL/getopt.c license is MIT, then please don't list it here: it's the same as the rest of Julia. Things are already complex enough regarding files which have a different license.

Not sure for NetBSD, but if that's exactly equivalent to MIT, maybe do not list it here either.